pkg/services/grafana-apiserver/auth/authorizer/provider.go

```diff
@@ -28,7 +28,11 @@ func ProvideAuthorizer(
 		authorizers = append(authorizers, orgIDAuthorizer)
 	}
 
-	authorizers = append(authorizers, orgRoleAuthorizer)
+	authorizers = append(authorizers,
+		orgRoleAuthorizer,
 
+		// Add this last so that if nothing says authorizer.DecisionDeny, it will pass
+		authorizerfactory.NewAlwaysAllowAuthorizer(),
```

Comment:
Do you expect to change this behavior eventually? It seems overly permissive. If it's just temporary and not meant to be long-term please add a TODO/FIXME comment to help us find it later?

Reply:
so no -- not a fixme.  The the alternative could be combining the logic from org_id|stack_id + org_role into one authorizer and saying that if nothing is Denied, then the request is OK.  I don't have any strong opinions if that makes it more or less clear

Reply:
I follow that. I still don't love it, but what's the current behavior? I'd expect that we always fall back to deny if there isn't an explicit allow, and we should implement the same behavior? I also get that EVERYTHING IS DIFFERENT) in this model - don't worry, I am not going to block this PR on this conversation.